lib/archethic_web/views/explorer_view.ex

```diff
@@ -196,5 +196,9 @@ defmodule ArchEthicWeb.ExplorerView do
     """
   end
 
+  def format_transaction_content(:origin_shared_secrets, content) do
+    Base.encode16(content)
```

Comment:
I was thinking more like decoding the binary to list the public keys

Reply:
Did it !